nextjs-13/src/lib/request/getServerSession.ts

```diff
@@ -0,0 +1,19 @@
+import { cookies, headers } from 'next/headers';
+import { getServerSession as originalGetServerSession } from 'next-auth';
+
+import { authOptions } from 'config/auth';
+
```

Comment:
Let's add a comment mentioning that it is a workaround and add a link to the source (https://github.com/nextauthjs/next-auth/issues/7486#issuecomment-1543747325)

Reply:
Added [here](https://github.com/nimblehq/js-framework-benchmark/pull/129/commits/64559b097e6d1a108ad3691bd16bb66eeb47a5ca)! 🙈